app/Helpers/database/achievement.php

```diff
@@ -483,7 +483,7 @@ function updateAchievementFlag(int|string|array $achID, int $newFlag): bool
         $updatedAchID = (int) $data['ID'];
         $updatedAchIDs[] = $updatedAchID;
 
-        $numUnlocks = getAchievementUnlockCount($updatedAchID);
+        $numUnlocks = Achievement::find($achID)->unlocks_total;
```

Comment:
It looks like this can be removed. It's only used to populate `$authorCount` and `$authorPoints`, which don't seem to be used.

Reply:
Nice, thanks! This didn't even occur to me.
Refactored `updateAchievementFlag()` slightly more in https://github.com/RetroAchievements/RAWeb/pull/1949/commits/b9771ac3635a646c913ae8e7a8b35d6e8010cb45